modules/social_features/social_group/social_group.module

```diff
@@ -317,10 +317,20 @@ function social_group_preprocess_group(array &$variables) {
       }
     }
   }
-
   // Count number of group members.
-  $group_members_count = \Drupal::service('social_group.group_members_count');
-  $variables['group_members'] = $group_members_count->getGroupMemberCount($group);
+  $variables['group_members'] = count($group->getMembers());
+
+  // Prepare variables for statistic block.
+  if ($group->getGroupType()->hasContentPlugin('group_node:event')) {
+    $variables['group_events'] = count($group->getContent('group_node:event'));
+  }
+  if ($group->getGroupType()->hasContentPlugin('group_node:topic')) {
+    $variables['group_topics'] = count($group->getContent('group_node:topic'));
+  }
```

Comment:
❗️ This is a serious performance regression. We're basically loading all the group content here before counting it. 

The code that was removed was more efficient. We should probably create a `GroupStatistics` service in the `social_group` module and create method for each type there. Then we can deprecate `SocialGroupMembersCount` and route the `SocialGroupMembersCount::getGroupMemberCount` call to `GroupStatistics::memberCount`.

Reply:
@Kingdutch 
I agree with you. I already fixed this performance issue [here](https://github.com/goalgorilla/open_social/pull/1515/commits/cd9ededd5a30a0f2fcfb176a63bc12b5972d5a7f).
I created a new service `GroupStatistics` which has 2 methods:
* `nodeCount` for an event and topic types
* `memberCount` for the `group_membership`
Also I removed old service `SocialGroupMembersCount` and used Dependency Injection in `SocialGroupListBuilder` for the `groupStatistics` service instead of `\Drupal->service()`